test/dummy/app/views/components/docs/test-component-with-duplicate-ids.yml

```diff
@@ -1,4 +1,7 @@
 name: Test invalid component with duplicate IDs
```

Comment:
Should this mention the image too? You're testing both it seems?

Reply:
Makes sense, will update